docs/v0.5.x/pipeline-hooks.md

```diff
@@ -99,6 +99,8 @@ didDeploy: --> runs at the end of a full deployment operation.
 teardown: ---> always the last hook being run
 ```
 
+### Additional hooks
+
 In addition, there are a few more specialized hooks that plugins may implement:
```

Comment:
this hook got here because it was in the rfc but I'm not sure if it ever got implemented

cannot find mention of it anywhere in the code

/cc @lukemelia